lib/paranoia.rb

```diff
@@ -219,8 +219,10 @@ def self.acts_as_paranoid(options={})
     def self.paranoia_scope
       where(paranoia_column => paranoia_sentinel_value)
     end
-    
-    unless options[:without_default_scope]
+
+    if options[:without_default_scope]
```

Comment:
We might as well add it even when `without_default_scope` is false.

Maybe someone will find use from `Model.unscoped.without_deleted`

Other than that LGTM :+1: